EyeSeeTea/home-page-app-dev
# Create landing page and button

### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/865ch8eed

### :memo: Implementation

- Not sure why we are using a nested array in `api/dataStore/home-page-app/landing-pages`, I've kept it for simplicity.
- Check the icons for the create actions in `CreateButton.tsx`

### :art: Screenshots

![image](https://github.com/EyeSeeTea/home-page-app-dev/assets/24643/e2bbabb0-d2a6-4c4d-a44c-879e447d9c41)

![image](https://github.com/EyeSeeTea/home-page-app-dev/assets/24643/f87bba8e-c8ed-413d-aa6a-141b2a565c4a)

### :fire: Testing

I've tested it with docker.eyeseetea.com/samaritans/dhis2-data:2.36.10.1-sp-cpr-dev-ministry-materials

- Create a new action.
- Create a new landing page.
- Check that the new landing page appears in the table after saving it.
- Check that both landing pages are visible for a specific user.
- Use sharing settings to show one or the landing page for that user.
